src/vs/workbench/api/common/extHostSpeech.ts

```diff
@@ -36,7 +36,11 @@ export class ExtHostSpeech implements ExtHostSpeechShape {
 		const cts = new CancellationTokenSource();
 		this.sessions.set(session, cts);
 
-		const speechToTextSession = disposables.add(provider.provideSpeechToTextSession(cts.token, language ? { language } : undefined));
+		const speechToTextSession = await provider.provideSpeechToTextSession(cts.token, language ? { language } : undefined);
```

Comment:
still add to disposables if defined?

Reply:
The session is no longer disposable, thats part of the PR.

Reply:
ah, I had checked the file's old version